lib/constructs/interface.js

```diff
@@ -172,6 +219,13 @@ Interface.prototype.generateExport = function () {
     this.setup(obj, constructorArgs, privateData);
     return utils.implForWrapper(obj);
   },
+  createDefaultIterator(target, kind) {
+    const iterator = Object.create(IteratorPrototype);
```

Comment:
I think Object.create is more correct actually, as it's not supposed to have a `.constructor`.